src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,58 @@
+# Common data types
+
+## Preprocessed or cleaned data
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Preprocessing in this context means transformations of data that do not change
+the data type of the input (as expressed by its BIDS `suffix`).
+For instance, a change in the number of dimensions is likely to disrupt the propagation
+of the input's `suffix` and generally, the outcomes of such transformation
+cannot be considered _preprocessed_ or _cleaned data_.
+Examples:
+
+ -  Motion-corrected, temporally denoised, and transformed to MNI space BOLD series
+ -  Inhomogeneity corrected and skull stripped T1w files
+ -  Motion-corrected DWI files
+ -  Time-domain filtered EEG data
+ -  Spatially filtered EEG data
```

Comment:
rather than `- Spatially filtered EEG data` which is not as clear as the `ICA cleaned EEG data` that it replaced, I recommend giving the example `- MaxFilter (e.g. SSS) cleaned MEG data`. That is more specific and broadens the examples to another modality.